### PR TITLE
Removed stop words from scoring

### LINKED ISSUE #250: Remove stopwords check for focus keyword
This check:

"The focus keyword for this page contains one or more stop words, consider removing them. Found 'of, the'."

Should become a grey bullet without a score. It should have the following text:

"Your focus keyword contains one or more stop words. This may or may not be wise depending on the circumstances. Read <a href="https://yoast.com/handling-stopwords/">this article</a> for more info."

If no stop words are founds within the focus keyword, nothing should be shown.

### PR DESCRIPTION
**Pull request by jcomack**
Stop words no longer count for the scoring, nor do they invoke a red indicator.

Fixes #250

### CODE REVIEW

File: spec/scoringSpec.js [hunk 1]
```diff
@@ -98,3 +102,19 @@ describe("A word count scoring", function() {
...
+          scores: [
+              { name: 'positiveScorer', score: 10 },
+              { name: 'negativeScorer', score: -100 },
+              { name: 'undefinedScorer', score: undefined }
+          ]
+      };
+
+       var formatter = new YoastSEO.ScoreFormatter(args);
+
+        expect(formatter.getUndefinedScores(args.scores) ).toContain( { name: 'undefinedScorer', score: undefined } );
```
Comment (atimmer) [inline spec/scoringSpec.js]: This test should be more specific. The following function would pass this test:
```
YoastSEO.ScoreFormatter.getUndefinedScores = function( scores ) {
    return scores;
}
```

File: js/scoreFormatter.js [hunk 2]
```diff
@@ -70,9 +72,26 @@ YoastSEO.ScoreFormatter.prototype.outputScore = function() {
...
+
 	this.scores = this.scores.sort( function( a, b ) {
 		return a.score - b.score;
 	} );
+
+	this.scores = unsortables.concat( this.scores );
+};
+
+/**
+ * Extracts scorers with a score of undefined
```
Comment (atimmer) [inline js/scoreFormatter.js]: Missing newline

File: js/scoreFormatter.js [hunk 3]
```diff
@@ -70,9 +72,26 @@ YoastSEO.ScoreFormatter.prototype.outputScore = function() {
  * sorts the scores array on ascending scores
  */
 YoastSEO.ScoreFormatter.prototype.sortScores = function() {
+	var unsortables = this.getUndefinedScores( this.scores );
```
Comment (atimmer) [inline js/scoreFormatter.js]: Doesn't this keep the undefined scores in the array duplicating all undefined scores?

**Comment by atimmer** [comment]
CR Done 👍🏻

**Comment by diedexx** [comment]
CR2 Done :+1:

**Comment by diedexx** [comment]
Acceptance done :+1:

**Pull request merged**